testcoverage.md

```diff
@@ -17,3 +17,23 @@ azure-data-explorer-datasource/ = 63.5%
 redshift-datasource/ = 48.6%
 timestream-datasource/ = 69.2%
 x-ray-datasource/ = 70.3%
+
+## Fri Mar 18 14:36:50 CET 2022
+### Frontend
+CloudWatch = 79.32%
+Cloud Monitoring = 49.52%
+Azure Monitor = 68.85%
+athena-datasource/ = 88.43%
+azure-data-explorer-datasource/ = 30.07%
+redshift-datasource/ = 85.59%
+timestream-datasource/ = 11.92%
```

Comment:
I noticed Timestream frontend coverage was 33.72% a few days ago but I don't see any major commits...

Reply:
So I just tried running tests on Timestream, and they were failing 🤷‍♀️  .
I had to nuke my `node_modules` folder and run `yarn install` again and it's indeed at `33.7%`

However I noticed that sometimes running `test:coverage` still outputs the coverage even if some tests are failing (for successful tests only, so that could been what happened for you?).

Reply:
```suggestion
timestream-datasource/ = 33.72%
```

Reply:
That's probably it. I'm going to remove the entry from the data which the grafana dashboard is reading from